cmd/test/cmd.go

```diff
@@ -235,9 +235,12 @@ func doRun(ctx context.Context, options *Options, ioCtrl *io.Controller, k8sLogg
 			DockerfileName:      "Dockerfile.test",
 			Deployable: deployable.Entity{
 				Commands: commands,
+				// Added this for backward compatibility. Before the refactor we were having the env variables for the external
+				// resources in the environment, so including it to set the env vars in the remote-run
+				External: manifest.External,
```

Comment:
We will need to see what is really needed here. In the case of `destroy`, I added it because those variables were available before, but they only have the default value. If during the deploy, some value was generated, they are not available, but should they?

Example of dynamic generation of external resources variables: https://github.com/ifbyol/node-compose-getting-started/blob/ifbyol/complete-manifest/okteto.yml#L48. Should they be available in test execution even if they were generated during deploy phase?

Reply:
That's a good point. +1 for removing since there's no need for backwards compatibility. @pchico83 thoughts?

Reply:
I'll merge to open a the next PR in the queue. I'll remove this is a follow up we if need be